src/styles/getMuiTheme.js

```diff
@@ -93,6 +104,9 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       width: spacing.desktopKeylineIncrement * 4,
       color: palette.canvasColor,
     },
+    list: {
```

Comment:
Will be subheader once https://github.com/callemall/material-ui/pull/2979 is merged.